src/core/client/stream/tabs/profile/components/HistoryComment.tsx

```diff
@@ -0,0 +1,69 @@
+import { Localized } from "fluent-react/compat";
+import * as React from "react";
+import { StatelessComponent } from "react";
+import Timestamp from "talk-stream/components/Timestamp";
+import {
+  BaseButton,
+  Flex,
+  HorizontalGutter,
+  Icon,
+  Typography,
+} from "talk-ui/components";
+import HTMLContent from "../../../components/HTMLContent";
+import * as styles from "./HistoryComment.css";
+
+export interface CommentHistoryProps {
+  comment: {
+    body: string | null;
+    createdAt: string;
+    replyCount: number | null;
+  };
+  goToConversation: () => void;
+}
+
+const HistoryComment: StatelessComponent<CommentHistoryProps> = props => {
+  return (
+    <HorizontalGutter>
+      <Flex direction="row" justifyContent="space-between">
+        <Typography variant="bodyCopy" headlineMapping={{ bodyCopy: "div" }}>
```

Comment:
Use `<Typography variant="bodyCopy" container="div">` instead. `headlineMapping` is more for setting defaults.